_examples/postcard/storage/watermill.go

```diff
@@ -0,0 +1,55 @@
+package storage
+
+import (
+	"database/sql"
+	"errors"
+
+	"github.com/google/uuid"
+
+	sql2 "github.com/ThreeDotsLabs/watermill-sql/pkg/sql"
+	"github.com/ThreeDotsLabs/watermill/message"
+)
+
+type WatermillSchemaAdapter struct{}
+
+func (s WatermillSchemaAdapter) SchemaInitializingQueries(topic string) []string {
+	return nil
+}
+
+func (s WatermillSchemaAdapter) InsertQuery(topic string, msgs message.Messages) (string, []interface{}, error) {
+	return "", nil, errors.New("not supported")
+}
+
+func (s WatermillSchemaAdapter) SelectQuery(topic string, consumerGroup string, offsetsAdapter sql2.OffsetsAdapter) (string, []interface{}) {
+	nextOffsetQuery, nextOffsetArgs := offsetsAdapter.NextOffsetQuery(topic, consumerGroup)
+	selectQuery := `
+		SELECT id, stream_id, event_payload FROM events
```

Comment:
I'm wondering if it wouldn't be possible to add extra config to the Watermill adapter to provide it instead of rewriting. 

It may be useful, as I suspect that such naive querying may not always work in Postgres properly. In that case, fixing it in our adapter will not propagate here.